src/types/onyx/TransactionViolation.ts

```diff
@@ -28,6 +28,7 @@ type TransactionViolation = {
         taxName?: string;
         tagListIndex?: number;
         tagListName?: string;
+        tagListValue?: string;
```

Comment:
We should remove this from transaction violations data, and pass this as a separate param to getErrorForField like we do for policyHasDependentTags. Let's call it `transactionTag`

Reply:
@cead22 Sadly `transactionTag` already exists in `MoneyRequestView.tsx`, so I named it tagValue instead. Is that fine to you? 😄